pkg/configtx/policies.go

```diff
@@ -18,12 +18,9 @@ import (
 	"github.com/hyperledger/fabric/common/policydsl"
 )
 
-// UpdateConsortiumChannelCreationPolicy update a consortium group's channel creation policy value
-func (c *ConfigTx) UpdateConsortiumChannelCreationPolicy(consortiumName string, policy Policy) error {
-	consortium, ok := c.updated.ChannelGroup.Groups[ConsortiumsGroupKey].Groups[consortiumName]
-	if !ok {
-		return fmt.Errorf("consortium %s does not exist in channel config", consortiumName)
-	}
+// SetConsortiumChannelCreationPolicy sets a consortium group's channel creation policy value
```

Comment:
This is the only function that my policy comment doesn't apply to

Reply:
done